src/YoutubeAtomPlaceholder.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+import { Placeholder } from './common/Placeholder';
+
+export const YoutubeAtomPlaceholder = (): JSX.Element => {
```

Comment:
Can we make this placeholder not shimmer and be black? I think that would be more in keeping with other media loading states

Perhaps we can also add a little transition so it doesn't appear quite so suddenly?

Reply:
Sure good idea, that will make the placeholder a lot more subtle.

I added a transition but commented out for now as it looks a bit jarring. Mainly because it doesn't appear for very long in the overlay->placeholder->player flow. The code is there though and we can experiment with it.

Reply:
Nice. And plus one for having a dedicated, inline placeholder